ops-tools/docker/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3"
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: ./images/app/Dockerfile
+    volumes:
+      - .:/app
+    ports:
+      - 8080:8080
+    depends_on:
+      - postgres
+      - redis
+
+  postgres:
+    image: "postgres"
+    restart: always
+    ports:
+      - 5432:5432
+    networks:
+      - docker-practice-net
+    volumes:
+      - ./data/postgres:/var/lib/postgresql/data
+      - ./data/dump:/db-dump
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: school
+      POSTGRES_DB: school
+
+  redis:
+    image: "redis:alpine"
```

Comment:
The same as above - we should specify versions of images which we use. It will guarantee stable work of our application.